src/app/public/styles/_mixins-public.scss

```diff
@@ -59,9 +59,9 @@
   }
 }
 
-@mixin sky-theme-mode-dark {
-  .sky-theme-mode-dark,
-  :host-context(.sky-theme-mode-dark) {
+@mixin sky-theme-modern-dark {
```

Comment:
I'm assuming nothing is consuming this mixin yet, hence the name change? I did a quick search but found nothing.

Reply:
Correct. Anything theme-related is essentially beta and subject to change. It turns out the original intent was for this mixin to be nested inside the `@sky-theme-modern`, but that turned out not to work, which is why it was renamed and its purpose changed to not be nested.

Reply:
Makes sense. Thanks for the explanation 👍